distributed/tests/test_worker.py

```diff
@@ -2199,7 +2200,7 @@ async def test_bad_executor_annotation(c, s, a, b):
 
 @gen_cluster(client=True)
 async def test_process_executor(c, s, a, b):
-    with ProcessPoolExecutor() as e:
+    with ProcessPoolExecutor(mp_context=multiprocessing.get_context("spawn")) as e:
```

Comment:
Might be best to use `distributed.utils.get_mp_context` which picks the context that is being configured by default. It also sets some preload things for a forkserver which isn't truly relevant. However, this would make the tests sensitive to what we're setting as defaults which sounds like a good thing to have

Reply:
Sure, can do that.